src/pages/LinkedAccounts/styles.scss

```diff
@@ -0,0 +1,54 @@
+.linked-accounts {
+  &__account {
+    border-bottom: 1px solid var(--text-secondary);
+    padding: 20px 0;
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+
+    &-header {
+      color: var(--text-primary);
+      font-weight: 600;
+
+      span {
+        font-weight: 400;
+        color: var(--text-secondary);
+      }
+    }
+
+    &-input {
+      display: flex;
+      flex-direction: column;
+      max-width: 600px;
+
+      input {
+        padding: 5px 10px;
+        border-radius: 12px;
+        background-color: white;
+        color: black;
+        outline: none;
+        border: 1px solid #060606;
+      }
+
+      span {
+        font-weight: 400;
+        color: var(--text-secondary);
+        font-size: 10px;
+      }
+    }
+
+    button {
+      width: 80px;
+      height: 20px;
+      font-size: 8px;
```

Comment:
This font is too small, do we need any overrides of the button size from the core button component?